pkg/controller/accountpool/accountpool_controller.go

```diff
@@ -138,6 +139,7 @@ func (r *ReconcileAccountPool) Reconcile(request reconcile.Request) (reconcile.R
 
 	// Create Account CR
 	newAccount := newAccountForCR(request.Namespace)
+	addFinalizer(newAccount, "finalizer.stable.example.com")
```

Comment:
@fahlmant should we make this `finzalizer.aws.managed.openshift.io`

Reply:
I'm +1 to that